consensus/istanbul/backend/pos.go

```diff
@@ -103,13 +103,15 @@ func (sb *Backend) distributeEpochRewards(header *types.Header, state *state.Sta
 		return err
 	}
 
-	// Validator rewards were paid in cUSD, convert that amount to cGLD and add it to the Reserve
-	totalValidatorRewardsConvertedToGold, err := currency.Convert(totalValidatorRewards, &stableTokenAddress, nil)
+	currencyManager := currency.NewManager(nil, nil)
+
+	// Validator rewards were paid in cUSD, convert that amount to CELO and add it to the Reserve
+	totalValidatorRewardsConvertedToCelo, err := currencyManager.ToCelo(totalValidatorRewards, &stableTokenAddress)
```

Comment:
Referencing https://github.com/celo-org/celo-blockchain/pull/1499#discussion_r612810987, I believe there is a hardfork here in the case that registry lookup fails for GoldToken contract

Reply:
removed that logic